source/components/StyledAvatar/DefaultAvatar.js

```diff
@@ -6,13 +6,13 @@ import IconAvatar from '../../icons/IconAvatar';
 
 // we have alt prop in rest
 // eslint-disable-next-line jsx-a11y/alt-text
-const DefaultAvatar = styled(({ borderWidth, ...rest }) => <IconAvatar {...rest} />)`
+const DefaultAvatar = styled(({ borderWidth, size, ...rest }) => <IconAvatar hight={size} width={size} {...rest} />)`
```

Comment:
Looks like spelling here should be 'height'.

Reply:
Fixed in 3d76415